reflex/utils/processes.py

```diff
@@ -145,3 +146,14 @@ def new_process(args, **kwargs):
         args,
         **kwargs,
     )
+
+
+def catch_keyboard_interrupt(signal, frame):
+    """Display a custom message with the current time when exiting an app.
+
+    Args:
+        signal: The keyboard interrupt signal.
+        frame: The current stack frame.
+    """
+    current_time = datetime.now().strftime("%H:%M:%S")
```

Comment:
```suggestion
    current_time = datetime.now().isoformat()
```

Reply:
Done.

Reply:
After running the code, the message becomes `Reflex app stopped at time: 2023-07-16T13:52:18.491325`, with a `T` in the middle.
Should I apply the format used when updating the app like here? 
`[Updating App] Applying changes and refreshing. Time: 2023-07-16 13:52:15.027191`

Reply:
Sure IMO.

Reply:
That T in the middle is the standard ISO8601 date format. I'm of the opinion that unless true localized date formats are used, then you should use ISO format to avoid ambiguity. I wouldn't do any further formatting to the date.